src/components/StudyRoom/StudyRoomCreate.js

```diff
@@ -30,11 +31,21 @@ export default function RoomCreation() {
         participants:[],
     });
 
-    React.useEffect(()=>{
-      let user = localStorage.getItem("username")
-        setRoomData({...roomData, owner:user});
-    })
+    React.useEffect(()=> {
+        let user = localStorage.getItem("username")
```

Comment:
JSON.parse(localStorage.getItem("email"));

Reply:
owner is the owner email.

Reply:
not in the scope of this PR